tox.ini

```
@@ -22,4 +22,5 @@ deps =
     django31: Django>=3.1,<3.2
     django32: Django>=3.2,<3.3
     django40: Django>=4.0,<4.1
+    django40: Django>=4.1,<4.2
```

Looks like a copy & paste error here:

```suggestion
    django41: Django>=4.1,<4.2
```

---

Oops, good catch